### PR TITLE
CASMNET-1156

### PR DESCRIPTION
**Pull request by lukebates123**
## Summary and Scope

Add Arista CHN peering to docs.

### CODE REVIEW

File: operations/network/customer_access_network/bi-can_arista_metallb_peering.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Metallb peering with arista edge router
```
Comment (nrockershousen) [inline operations/network/customer_access_network/bi-can_arista_metallb_peering.md:1]: Change to:

MetalLB Peering with Arista Edge Router

File: operations/network/customer_access_network/bi-can_arista_metallb_peering.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+This is an example configuration of how to connect a pair of Arista switches to metallb running inside of kubernetes.
+
+## Prerequisites
+
+- Pair of arista switches already connected to the high speed network.
```
Comment (nrockershousen) [inline operations/network/customer_access_network/bi-can_arista_metallb_peering.md:7]: - Capitalize Arista
- Add a hyphen to high-speed

File: operations/network/customer_access_network/bi-can_arista_metallb_peering.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
+# Metallb peering with arista edge router
+
+This is an example configuration of how to connect a pair of Arista switches to metallb running inside of kubernetes.
```
Comment (nrockershousen) [inline operations/network/customer_access_network/bi-can_arista_metallb_peering.md:3]: metallb -> MetalLB
kubernetes -> Kubernetes

File: operations/network/customer_access_network/bi-can_arista_metallb_peering.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+## Prerequisites
+
+- Pair of arista switches already connected to the high speed network.
+- Updated SLS file that has the `CHN` network configured.
```
Comment (nrockershousen) [inline operations/network/customer_access_network/bi-can_arista_metallb_peering.md:8]: Spell out SLS here:

System Layout Service (SLS)

File: operations/network/customer_access_network/bi-can_arista_metallb_peering.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+```json
+                "IPAddress": "10.103.9.2",
+                "Name": "chn-switch-1"
+              },
+              {
+                "IPAddress": "10.103.9.3",
+                "Name": "chn-switch-2"
+```
+
+Below will be the config needed on both switches.
```
Comment (nrockershousen) [inline operations/network/customer_access_network/bi-can_arista_metallb_peering.md:106]: Change to:

The following config is needed on both switches:

File: operations/network/customer_access_network/bi-can_arista_metallb_peering.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+                ],
+                "Comment": "x3000c0s4b0n0",
+                "IPAddress": "10.103.9.10",
+                "Name": "ncn-w001"
+              },
+```
+
+In this example, `chn-switch-1` and `chn-switch-2` will be the Arista pair.
+
+SLS entries from the above output
```
Comment (nrockershousen) [inline operations/network/customer_access_network/bi-can_arista_metallb_peering.md:95]: add `:` character to end of line

File: operations/network/customer_access_network/bi-can_arista_metallb_peering.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+                "Name": "chn-switch-2"
+```
+
+Below will be the config needed on both switches.
+
+- The prefix list will be the subnet of the CHN, the `ge` will equal the cidr.
+  - This prevents routes from other networks being installed into the routing table.
+
+- `router bgp 65533` will match the ASN from sls.`"MyASN": 65533,`
+- the `neighbor` will match every worker node.
```
Comment (nrockershousen) [inline operations/network/customer_access_network/bi-can_arista_metallb_peering.md:112]: Capitalize "the"

File: operations/network/customer_access_network/bi-can_arista_metallb_peering.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+                "IPAddress": "10.103.9.3",
+                "Name": "chn-switch-2"
+```
+
+Below will be the config needed on both switches.
+
+- The prefix list will be the subnet of the CHN, the `ge` will equal the cidr.
+  - This prevents routes from other networks being installed into the routing table.
+
+- `router bgp 65533` will match the ASN from sls.`"MyASN": 65533,`
```
Comment (nrockershousen) [inline operations/network/customer_access_network/bi-can_arista_metallb_peering.md:111]: "sls" should be uppercase

File: operations/network/customer_access_network/bi-can_arista_metallb_peering.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
+# Metallb peering with arista edge router
+
+This is an example configuration of how to connect a pair of Arista switches to metallb running inside of kubernetes.
+
+## Prerequisites
+
+- Pair of arista switches already connected to the high speed network.
+- Updated SLS file that has the `CHN` network configured.
+
```
Comment (nrockershousen) [inline operations/network/customer_access_network/bi-can_arista_metallb_peering.md:9]: Add a new header here to separate from prereqs. Maybe... `## Example Configuration`

File: operations/network/management_network/index.md [hunk 10]
```diff
@@ -132,6 +132,7 @@ and simplified Spanning-tree configuration. New default configuration:
 - Spanning-tree admin-edge port settings are retained for allowing quicker PXE boot
 - Spanning-tree instance is tied to default (no longer multiple instances or per vlan) 
 - Spanning-tree MSTP configuration name and revision in all configurations generated by CANU are set
+- Add peering from metallb to customer edge router.
```
Comment (nrockershousen) [inline operations/network/management_network/index.md:135]: metallb -> MetalLB

**Comment by gbaker-hpe** [comment]
@nrockershousen In the future, you could also update the PR and make the changes directly.

**Comment by nrockershousen** [comment]
> Thanks @nrockershousen. These look good. Since the changes are small, does it make sense for similar suggest to be direct edits by the reviewer? Either way works for us.

That sounds good to me. I can make that kind of change directly.

**Review by gbaker-hpe — APPROVED** [review]

**Comment by gbaker-hpe** [comment]
Thanks @nrockershousen this looks great!

**Pull request merged**